// mpvue-entry/dist/pagesListIndex.js
import Vue from 'vue'

import App from '@/pages/list/index'

import MpvueRouterPatch from 'mpvue-router-patch';

Vue.use(MpvueRouterPatch);

import '@/wux/styles/index.wxss';

Vue.prototype.ajax = function (url, data, method) {

  return new Promise((resolve, reject) => {

    wx.request({

      //url: `https://nhjk.uniteddata.com/${url}`,

      url: `http://172.16.1.117:5000/${url}`,

      data: data,

      method: method ? 'POST' : 'GET',

      header: {

        'Authorization': wx.getStorageSync('Authorization'),

        'Content-Type': 'application/x-www-form-urlencoded'

      },

      success(res) {

        resolve(res.data);

      },

      fail(err) {

        reject(err);

      }

    });

  })

}

Vue.config.productionTip = false

const app = new Vue({

  ...App

})

app.$mount();

export default {
  "navigationBarTitleText": "设备列表",
  "usingComponents": {
    "wux-toast": "../../wux/toast/index",
    "wux-row": "../../wux/row/index",
    "wux-col": "../../wux/col/index",
    "wux-grids": "../../wux/grids/index",
    "wux-grid": "../../wux/grid/index",
    "wux-icon": "../../wux/icon/index",
    "wux-button": "../../wux/button/index",
    "wux-search-bar": "../../wux/search-bar/index"
  }
}
