// mpvue-entry/dist/pagesIndexIndex.js
import Vue from 'vue'

import App from '@/pages/index/index'

import MpvueRouterPatch from 'mpvue-router-patch';

Vue.use(MpvueRouterPatch);

import { $wuxToast } from "D:\\git\\monitor\\static\\wux\\index";

import 'D:\\git\\monitor\\static\\wux\\styles\\index.wxss';

// Vue.prototype.$url = 'http://172.16.0.167:5000'//测试接口

// Vue.prototype.$wss = 'ws://172.16.0.167:5005'//测试接口

Vue.prototype.$url = 'https://nhjk.uniteddata.com'//线上接口

Vue.prototype.$wss = 'wss://nhjk.uniteddata.com/wss'//线上接口

Vue.prototype.ajax = function (url, data, method) {

  return new Promise((resolve, reject) => {

    wx.request({

      url: `${this.$url}/${url}`,

      data: data,

      method: method ? method : 'GET',

      header: {

        'Authorization': wx.getStorageSync('Authorization'),

        'Content-Type': 'application/x-www-form-urlencoded'

      },

      success(res) {

        // console.log(res.data);

        resolve(res.data);

      },

      fail(err) {

        reject(err);

      }

    });

  })

}

Vue.prototype.Toast = function (type, text) {

  $wuxToast().show({

    type: type,

    duration: 1500,

    color: "#ffffff",

    text: text

  });

}

Vue.filter('Rounding', function (value) {

  value = Number(value);

  return value.toFixed(0);

})

Vue.config.productionTip = false

const app = new Vue({

  ...App

})

app.$mount();

export default {
  "navigationBarTitleText": "统计分析",
  "usingComponents": {
    "wux-row": "/static/wux/row/index",
    "wux-col": "/static/wux/col/index",
    "wux-icon": "/static/wux/icon/index",
    "wux-select": "/static/wux/select/index",
    "ff-canvas": "/static/f2-canvas/f2-canvas",
    "wux-loading": "/static/wux/loading/index"
  }
}
