// mpvue-entry/dist/pagesLoginIndex.js
import Vue from 'vue'

import App from '@/pages/login/index'

import MpvueRouterPatch from 'mpvue-router-patch';

Vue.use(MpvueRouterPatch);

import { $wuxToast } from "@/wux/index";

import '@/wux/styles/index.wxss';

Vue.prototype.$url = 'http://172.16.1.117:5000'//测试接口

Vue.prototype.$wss = 'ws://172.16.1.117:5005'//测试接口

// Vue.prototype.$url = 'https://nhjk.uniteddata.com'//线上接口

// Vue.prototype.$wss = 'wss://nhjk.uniteddata.com'//线上接口

Vue.prototype.ajax = function (url, data, method) {

  return new Promise((resolve, reject) => {

    wx.request({

      url: `${this.$url}/${url}`,

      data: data,

      method: method ? 'POST' : 'GET',

      header: {

        'Authorization': wx.getStorageSync('Authorization'),

        'Content-Type': 'application/x-www-form-urlencoded'

      },

      success(res) {

        console.log(res);

        resolve(res.data);

      },

      fail(err) {

        reject(err);

      }

    });

  })

}

Vue.prototype.Toast = function (type, text) {

  $wuxToast().show({

    type: type,

    duration: 1500,

    color: "#ffffff",

    text: text

  });

}

Vue.filter('Temperature', function (value) {

  value = Number(value);

  return value.toFixed(1);

});

Vue.config.productionTip = false

const app = new Vue({

  ...App

})

app.$mount();

export default {
  "navigationBarTitleText": "登录",
  "usingComponents": {
    "wux-button": "../../wux/button/index"
  }
}
